src/Car.java

```diff
@@ -0,0 +1,15 @@
+public class Car extends Vehicle {
+
+    public Car(String modelName, int wheelsCount) {
+        super(modelName, wheelsCount);
+    }
+
+    @Override
+    public void check() {
+        super.check();
+        checkEngine();
+    }
+    private void checkEngine() {
+        System.out.println("Проверяем двигатель " + modelName);
```

Comment:
нужно уточнять реализацию, чтобы был виден полиморфизм
```suggestion
        System.out.println("Проверяем двигатель машины " + modelName);
```